### ISSUE TITLE
안녕하세요. ^^ 

### ISSUE THREAD

**Issue by mjm4sky** [axisj/axisj#923]
var fnObj = {
		pageStart: function(){
			fnObj.searchInit();
			fnObj.grid.init();
		},
		pageResize: function(){

		},
		searchInit: function(){
			mySearch.setConfig({
				targetID:"AXSearchTarget",
				theme : "AXSearch",
				rows:[
					{display:true, addClass:"", style:"", list:[
						{label:"", labelWidth:"", type:"selectBox", width:"100", key:"srh_state", addClass:"secondItem", valueBoxStyle:"", value:"",
							options:[ 
								{optionValue:'1', optionText:'일반'}, 
								{optionValue:'2', optionText:'탈퇴'} 
							]
						},
						{label:"", labelWidth:"", type:"selectBox", width:"100", key:"searchBy", addClass:"secondItem", valueBoxStyle:"", value:"",
							options:[ 
								{optionValue:'userId', optionText:'아이디'}, 
								{optionValue:'depositors', optionText:'이름'}, 
								{optionValue:'nickName', optionText:'닉네임'}
							]
						},
						{label:"", labelWidth:"", type:"inputText", width:"100", key:"searchWord", addClass:"", valueBoxStyle:"", value:""},

						{label:"", labelWidth:"", type:"button", width:"80", key:"button", addClass:"", valueBoxStyle:"padding-left:0px;padding-right:5px;", value:"검색하기",
							onclick: function(){
								fnObj.grid.search();
							}
						}
					]}
				]
			});
		},
		
		grid: {
			init: function () {
				myGrid.setConfig({
					emptyListMSG: "데이터가 존재하지 않습니다.",
					targetID: "AXGridTarget",
					colHeadAlign: "center",
					colHeadTool : false,
					theme: "AXGrid",
					colGroup: [
						{ key: "m_ID", label: "checkbox", width: "30", align: "center", formatter: "checkbox", checked: function(){ } },
						{ key: "userId", label: "아이디(닉네임)", width: "150", align: "center", formatter: function() {
							var userId = this.item.userId + '(' + this.item.nickName + ')';
							return userId;
						} },
						{ key: "depositors", label: "이름", width: "100", align: "center", formatter: "dec" },
						{ key: "mGrade", label: "등급", width: "80", align: "center" },
						{ key: "recommend_CNT", label: "추천수", width: "70", align: "right" }
					],
  
					body: {
						onclick: function () {
						}
					},
					page:{
						paging:true,
						onchange: function(pageNo){
							trace(this, pageNo);
						}
					}
				});

				fnObj.grid.search();
			},
			
			search: function () {
				var pars = mySearch.getParam();

				new AXReq("member_list_ajax.asp", {
					method: "POST", 
					pars: pars,
					onsucc:function(res){
						var data = {
							list: res.data,
							page:{
								pageNo: res.pageNo, // 현재페이지
								pageSize: res.pageSize,
								pageCount: res.pageCount, // 전체 페이지수
								listCount: res.listCount // 우측하단 카운트 표시 갯수
							}
						};
						myGrid.setData(data);
					}
				});
			
				return false;
			}
		};
